org.elysium.parent/org.elysium.ui/src/org/elysium/ui/compiler/CompilerConsole.java

```diff
@@ -102,6 +102,15 @@ public static CompilerConsole get(IFile file) {
 		return result;
 	}
 
+	/**
+	 * Returns the console for the syntax update action.
+	 */
+	public static CompilerConsole getSyntaxUpdateConsole() {
```

Comment:
I know the location of these static methods is somewhat arbitrary. If you'd agree, move this method to `SyntaxUpdaterJob` and `CompilerConsole.get` to `CompilerJob`. But I don't want to bikeshed. :)